packages/graphql/src/utils/utils.ts

```diff
@@ -28,3 +28,30 @@ export function isString(value: unknown): value is string {
 export function isNeoInt(value: unknown): value is Integer {
     return isInt(value);
 }
+
+/** Joins all strings with given separator, ignoring empty or undefined statements */
+export function joinStrings(statements: string | Array<string | undefined>, separator = "\n"): string {
+    return filterTruthy(arrayfy(statements)).join(separator);
+}
+
+/** Makes sure input is an array, if not it turns into an array (empty array if input is null or undefined) */
+export function arrayfy<T>(raw: T | Array<T> | undefined | null): Array<T> {
```

Comment:
Mmm, agree that this may not be the best name, but assert may be confusing, as an element is converted into an array if it is not, instead of failing.

what about `asArray` or `toArray`?

Reply:
Either of those sound good!